epub33/fxl-a11y-tech/index.html

```diff
@@ -292,6 +292,65 @@ <h3>Media Overlays</h3>
+                <aside class="example" title="Accessible table example">
+                    <pre>
+            &lt;table&gt;
+                &lt;caption&gt;Sample Data Table&lt;/caption&gt;
+                &lt;thead&gt;
+                    &lt;tr&gt;
+                        &lt;th scope="col"&gt;Header 1&lt;/th&gt;
+                        &lt;th scope="col"&gt;Header 2&lt;/th&gt;
+                        &lt;th scope="col"&gt;Header 3&lt;/th&gt;
+                    &lt;/tr&gt;
+                &lt;/thead&gt;
+                &lt;tbody&gt;
+                    &lt;tr&gt;
+                        &lt;th scope="row"&gt;Row 1&lt;/th&gt;
+                        &lt;td&gt;Data 1&lt;/td&gt;
+                        &lt;td&gt;Data 2&lt;/td&gt;
+                    &lt;/tr&gt;
+                    &lt;tr&gt;
+                        &lt;th scope="row"&gt;Row 2&lt;/th&gt;
+                        &lt;td&gt;Data 3&lt;/td&gt;
+                        &lt;td&gt;Data 4&lt;/td&gt;
+                    &lt;/tr&gt;
+                &lt;/tbody&gt;
+            &lt;/table&gt;
+                    </pre>
+                </aside>
+                <p>Complex tables can be authored by using additional attributes and elements to ensure that merged cells and other complex structures are understandable to all users. Proper use of headers, scope, and caption is then essential. For example:</p>
+                <aside class="example" title="Complex accessible table example">
+                    <pre>
+            &lt;table&gt;
+                &lt;caption&gt;Complex Data Table with Merged Cells&lt;/caption&gt;
+                &lt;thead&gt;
+                    &lt;tr&gt;
+                      &lt;th scope="col"&gt;Header 1&lt;/th&gt;
+                      &lt;th scope="col"&gt;Header 2&lt;/th&gt;
+                      &lt;th scope="col"&gt;Header 3&lt;/th&gt;
+                    &lt;/tr&gt;
+                &lt;/thead&gt;
+                &lt;tbody&gt;
+                    &lt;tr&gt;
+                      &lt;th scope="row" id="row1"&gt;Row 1&lt;/th&gt;
+                      &lt;td headers="row1 header2"&gt;Data 1&lt;/td&gt;
+                      &lt;td headers="row1 header3"&gt;Data 2&lt;/td&gt;
+                    &lt;/tr&gt;
+                    &lt;tr&gt;
+                      &lt;th scope="row" id="row2"&gt;Row 2&lt;/th&gt;
+                      &lt;td headers="row2 header2" rowspan="2"&gt;Merged Data&lt;/td&gt;
+                      &lt;td headers="row2 header3"&gt;Data 3&lt;/td&gt;
+                    &lt;/tr&gt;
+                    &lt;tr&gt;
+                      &lt;th scope="row" id="row3"&gt;Row 3&lt;/th&gt;
+                      &lt;td headers="row3 header3"&gt;Data 4&lt;/td&gt;
+                    &lt;/tr&gt;
+                &lt;/tbody&gt;
+            &lt;/table&gt;
+                    </pre>
+                 </aside>
```

Comment:
Would it be a good idea to actually render what the table would look like below the code example?